sites/main-site/src/content/hackathon-projects/hackathon-march-2025/bioimage-OME-tools.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Bioimage processing modules based on OME tools
+category: components
+intro_video: ""
+image: "/assets/images/events/2025/hackathon-march/bioimage_meme.png"
+image_alt: "Heading: "
```

Comment:
Please write a real alt text for the image https://accessibility.huit.harvard.edu/describe-content-images see other projects

Reply:
Thanks for the feedback! we indeed forgot about it. We added a description for image, hopefully it’s compliant